pkg/kubectl/cmd/run.go

```diff
@@ -83,7 +86,8 @@ func NewCmdRun(f *cmdutil.Factory, cmdIn io.Reader, cmdOut, cmdErr io.Writer) *c
 }
 
 func addRunFlags(cmd *cobra.Command) {
-	cmd.Flags().String("generator", "", "The name of the API generator to use.  Default is 'run/v1' if --restart=Always, otherwise the default is 'run-pod/v1'.")
+	// TODO: Change the default to "deployment/v1beta1" when deployment reaches beta (#15313)
```

Comment:
Yes, and this flag description needs to be updated too. I want to put this TODO in both places just in case people change the behavior without updating flag descriptions.